From 522676e682826a53422081e1391d149376da48be Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Wed, 7 Mar 2018 00:06:37 +0000
Subject: [PATCH] OS-6178 Support "routes" for KVM and bhyve instances Reviewed
 by: Dan McDonald <danmcd@joyent.com> Approved by: Josh Wilsdon
 <josh@wilsdon.ca>

---
 src/vm/node_modules/proptable.js | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/vm/node_modules/proptable.js b/src/vm/node_modules/proptable.js
index 4092fcd0..7bbb6c5e 100644
--- a/src/vm/node_modules/proptable.js
+++ b/src/vm/node_modules/proptable.js
@@ -1619,8 +1619,10 @@ exports.properties = {
         json: 'routes',
         payload: {
             allowed: {
+                'bhyve': ['create', 'receive'],
                 'joyent': ['create', 'receive'],
                 'joyent-minimal': ['create', 'receive'],
+                'kvm': ['create', 'receive'],
                 'lx': ['create', 'receive']
             },
             type: 'flat-object'
@@ -1635,8 +1637,10 @@ exports.properties = {
     }, set_routes: {
         payload: {
             allowed: {
+                'bhyve': ['update'],
                 'joyent': ['update'],
                 'joyent-minimal': ['update'],
+                'kvm': ['update'],
                 'lx': ['update']
             },
             type: 'flat-object'
-- 
2.21.0

